app/backend/templates/v2/event_create.mjml

```diff
@@ -30,6 +30,6 @@
 </mj-section>
 <mj-section padding="0px">
   <mj-column>
-    <mj-text><b>Do not reply to this email.</b> Use one of the buttons above, or log in to your account to reply to this request.</mj-text>
+    <mj-text>Best,<br />The Couchers.org Team</mj-text>
```

Comment:
:see_no_evil:

Reply:
Haha, well it was a lot of text you needed to write - I'd say you are excused 😉